package-lock.json

```diff
@@ -0,0 +1,2190 @@
+{
```

Comment:
This should not be included as we are using yarn and not npm to manage our packages. Please remove again :)

Reply:
I know. There is another file too which should not be included into the commit. I'll take care of this